tests/integration/acp/p2p/create_test.go

```diff
@@ -130,3 +130,147 @@ func TestACP_P2PCreatePrivateDocumentsOnDifferentNodes_SourceHubACP(t *testing.T
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestACP_P2PCreatePrivateDocumentsOnDifferentNodes_SourceHubACP1(t *testing.T) {
```

Comment:
todo: adjust the name of the test. 

It doesn't create private documents on different nodes. Just 1 private doc on one node.

Reply:
Thanks for flagging this. I totally forgot to rename after copying.

Reply:
✅